app/src/main/java/org/ole/planet/myplanet/repository/DiagnosticsRepositoryImpl.kt

```
@@ -54,8 +54,7 @@ class DiagnosticsRepositoryImpl @Inject constructor(
             val log = buildApkLog(sharedPrefManager, model?.id, time, type, error)
             apkLogDao.insert(log)
             true
-        } catch (e: Exception) {
-            e.printStackTrace()
+        } catch (e: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)